### PR TITLE
Added examples for R API

### PR DESCRIPTION
**Pull request by falaki**
With spark 1.4 we can user spark-csv in R.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,5 +123,16 @@ df = sqlContext.load(source="com.databricks.spark.csv", header="true", path = "c
 df.select("year", "model").save("newcars.csv", "com.databricks.spark.csv")
 ```
 
+### R API
+Spark 1.4+:
+```R
+sc <- sparkR.init()
```
Comment (davies) [inline README.md]: This line is not necessary, sparkRSQL.init() will do this.

**Comment by davies** [comment]
LGTM, just one minor comment.

**Comment by davies** [comment]
@falaki The `schema` in read.df does not catch 1.4 release, it's only in 1.4.1 or 1.5

**Comment by falaki** [comment]
I see. Thanks for noting. I will remove that reference for now.

**Pull request merged**